scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
```

Comment:
:stop_sign: **Security Vulnerability**: SQL injection vulnerability pattern detected. This allows potential attackers to manipulate or extract data from the database[^1].

[^1]: CWE-89: Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection') - https://cwe.mitre.org/data/definitions/89.html
```suggestion
// Use parameterized queries to prevent SQL injection
const query = 'SELECT * FROM users WHERE id = ?'
const params = [process.argv[2]]

// Assuming you're using a database library like mysql2
connection.execute(query, params, (err, results) => {
    if (err) throw err
    console.log(results)
})
```